include/deal2lkit/sundials_interface.h

```diff
@@ -38,18 +58,25 @@ public :
 
 
   /**
-   * Create a compatible vector with the domain of F.
+   * Create a compatible vector with the domain of @f$ F @f$.
    */
   virtual shared_ptr<VEC>
   create_new_vector() const = 0;
 
-  /** Returns the number of degrees of freedom. Pure virtual function. */
+  /**
+   * This is a pure virtual function used to get the number of degrees of
+   * freedom.
+   */
   virtual unsigned int n_dofs() const = 0;
 
-  /** This function is called at the end of each iteration step for
-   * the ode solver. Once again, the conversion between pointers and
+  /**
+   * This function is called at the end of each iteration step of
+   * the ODE solver.
```

Comment:
I am referring to the ODE solver not to the system solver. 
I will try to be clearer.

> On 24 Sep 2015, at 10:34, Alberto Sartori <notifications@github.com> wrote:
> 
> In include/deal2lkit/sundials_interface.h <https://github.com/mathLab/deal2lkit/pull/138#discussion_r40296098>:
> 
> >    virtual unsigned int n_dofs() const = 0;
> >  
> > -  /** This function is called at the end of each iteration step for
> > -   * the ode solver. Once again, the conversion between pointers and
> > +  /**
> > +   * This function is called at the end of each iteration step of
> > +   * the ODE solver.
> Not sure if iteration step is the right name. It is called at every time interval, which is given by the user. Between two output steps, the system is solved as many time as IDA decides
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/mathLab/deal2lkit/pull/138/files#r40296098>.
>